app/test_main.py

```diff
@@ -1,3 +1,16 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
```

Comment:
Add tests for negative numbers and incorrect types

Reply:
Do I need to check the numbers that a function takes or returns?